sdk/avalon_sdk/ethereum/truffle_artifacts/truffle-config.js

```diff
@@ -54,6 +54,13 @@ module.exports = {
             gas: 4500000,
             gasPrice: 0
        },
+       avalon_docker: {
+            host: "172.13.0.3",
+            network_id: "*",
+            port: 8555,
+            gas: 4500000,
+            gasPrice: 0
+       },
        develop: {
             host: "10.66.245.80",     // Localhost (default: none)
```

Comment:
Can we use localhost instead of hard coded value?

Reply:
Localhost cannot be used. Moreover this is meant for Ganache that would be updated when enabling Ganache.

Reply:
If localhost cannot be used, you need to explain that in the comment. Otherwise someone will replace the hard-coded 10.66.245.80 with localhost or 127.0.0.1.